packages/asc-ui/src/utils/hooks/useFocus.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useRef, useState } from 'react'
+import Button from '../../components/Button'
+import useFocus from './useFocus'
```

Comment:
Can you add a README for this? Maybe explain the difference between the `useFocus()` function and `autofocus` prop

Reply:
There is a story that describes how to use this, and the function itself is documented by JSDoc. That should be enough IMHO